src/main/java/scottie/Scottie.java

```diff
@@ -0,0 +1,34 @@
+package scottie;
```

Comment:
All of your variable and method names are very intuitive and does not deviate from the coding standard! Nicely done